shopgate/pwa
# Added flushTab option for legacy links within the parsed-link helper

# Description
This ticket is about to fix an issue that occurred on iOS when the user opens multiple legacy pages after each other. In that case a double animation comes visible where one sees the previous opened page sliding out.

## Type of change

- [  ] New Feature :rocket: (non-breaking change which adds functionality)
- [  ] Breaking Change :boom: (fix or feature that would cause existing functionality to not work as expected)
- [  ] Polish :nail_care: (Just some cleanups)
- [  ] Docs :memo: (Changes in the documentations)
- [  ] Internal :house: Only relates to internal processes.

## How to test it
Open the legacy checkout twice after each other. With the fix of this ticket you shouldn't see a double animation anymore.
